packages/wonder-blocks-icon/docs.md

```diff
@@ -0,0 +1,135 @@
+
+```jsx
+const {StyleSheet} = require("aphrodite");
+
+const Color = require("@khanacademy/wonder-blocks-color").default;
+const {View} = require("@khanacademy/wonder-blocks-core");
+const Icon = require("@khanacademy/wonder-blocks-icon").default;
+const iconPaths = require("@khanacademy/wonder-blocks-icon/util/icon-paths.js").default;
+const size = "small";
+const iconNames = Object.keys(iconPaths).filter(name => iconPaths[name][size]);
+
+const IconContainer = (props) => (
+    <View style={styles.iconContainer}>
+        {props.icon}
+        {props.name}
+    </View>
+)
+
+const styles = StyleSheet.create({
+    iconContainer: {
+        alignItems: "center",
+        border: "2px solid",
+        borderColor: Color.blue,
+        borderRadius: 4,
+        color: Color.blue,
+        display: "inline-flex",
+        flexDirection: "row",
+        lineHeight: "16px",
+        margin: 2,
+        padding: 4,
+    },
+    icon: {
+        marginRight: 4,
+    },
+});
+
+<div>
+    {iconNames.map((name, i) => <IconContainer
+        key={i}
+        icon={<Icon
+            color={Color.blue}
+            name={name}
+            size={size}
+            style={styles.icon}
+            aria-label={`The ${name} icon`}
+        />}
+        name={name}
+    />)}
+</div>
+
+```
+
+All of the `medium` icons by name:
+
+```jsx
+const {StyleSheet} = require("aphrodite");
+
+const Color = require("@khanacademy/wonder-blocks-color").default;
+const {View} = require("@khanacademy/wonder-blocks-core");
+const Icon = require("@khanacademy/wonder-blocks-icon").default;
+const iconPaths = require("@khanacademy/wonder-blocks-icon/util/icon-paths.js").default;
+const size = "medium";
+const iconNames = Object.keys(iconPaths).filter(name => iconPaths[name][size]);
+
+const IconContainer = (props) => (
+    <View style={styles.iconContainer}>
+        {props.icon}
+        {props.name}
+    </View>
+)
+
+const styles = StyleSheet.create({
+    iconContainer: {
+        alignItems: "center",
+        border: "2px solid",
+        borderColor: Color.blue,
+        borderRadius: 4,
+        color: Color.blue,
+        display: "inline-flex",
+        flexDirection: "row",
+        lineHeight: "24px",
+        margin: 4,
+        padding: 4,
+    },
+    icon: {
+        marginRight: 4,
+    },
+});
+
+<div>
+    {iconNames.map((name, i) => <IconContainer
+        key={i}
+        icon={<Icon
+            color={Color.blue}
+            name={name}
+            size={size}
+            style={styles.icon}
+            aria-label={`The ${name} icon`}
+        />}
+        name={name}
+    />)}
+</div>
+
+```
+
+While we don't currently have assets for sizes larger than `medium`, we can
+still render the icons at `large` and `xlarge` in a pinch.
+```jsx
+
+const Icon = require("@khanacademy/wonder-blocks-icon").default;
+const iconPaths = require("@khanacademy/wonder-blocks-icon/util/icon-paths.js").default;
+
+<Icon size="large" name="dismiss" />
```

Comment:
I would move this an the next example up to the top since these examples are easy to understand as well as copy and paste.

Reply:
The dismiss icon looks like the letter X, maybe a different icon that looks more icon-y.

Reply:
👍